front-end/LoanStaticButton/LoanStaticButtonStyle.js

```diff
@@ -0,0 +1,24 @@
+import { StyleSheet } from 'react-native'
+
+const style = StyleSheet.create({
```

Comment:
@JialinWang1 and for the FloatingActionButton as well:
https://docs.nativebase.io/Components.html#fabs-def-headref